### ISSUE TITLE
Subsonic a service requires manual db deletion

### ISSUE THREAD

**Issue by Fresheyeball** [NixOS/nixpkgs#25503]
## Issue description
When enabling `services.subsonic.enable = true;` in `configuration.nix`. The service does not turn on properly. Instead, we get a 503 on `localhost:4040`, with jetty reporting an unavailable service.

This can be fixed by
```bash
sudo systemctl stop subsonic
sudo su
rm -rf /var/lib/subsonic/db
exit
sudo systemctl start subsonic
```
Subsonic will make a new db. And it will now work fine.

### Steps to reproduce
Add `services.subsonic.enable = true;` to `configuration.nix`. Ensure its running with `sudo systemctl start subsonic`. Open `localhost:4040` in your browser. Observe subsonic is not running.

## Technical details

* System: 17.03.873.0c041520c3 (Gorilla)
* Nix version: nix-env (Nix) 1.11.8
* Nixpkgs version: "17.03.873.0c041520c3"

**Comment by Fresheyeball** [comment]
@telotortium

**Comment by telotortium** [comment]
Hi, I can't reproduce this. Can you reliably reproduce this?

**Comment by Fresheyeball** [comment]
@telotortium yes. And I learned today the problem comes back if I restart the computer.

**Comment by fpletz** [comment]
Did you check the log for error messages? `journalctl -u subsonic`

**Comment by Fresheyeball** [comment]
@fpletz Yes, that's how I determined the db was the problem. I think its permissions.

**Comment by Fresheyeball** [comment]
@telotortium is there any way you can jump on a call with me? This problem just keeps persisting for me. I don't know how I can describe it better.

**Comment by telotortium** [comment]
I live in California. What time works best for you?

**Comment by pSub** [comment]
@Fresheyeball do you still have the problem? I am running a subsonic server for quite some time and never experienced this issue.

**Comment by Ekleog** [comment]
(triage) No answer about whether the OP still has the issue, no one able to reproduce, I suggest closing?
